internal/consensus/state_test.go

```diff
@@ -1582,91 +1582,114 @@ func TestStateLock_POLSafety2(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	cs1, vss := randState(4)
-	vs2, vs3, vs4 := vss[1], vss[2], vss[3]
-	height, round, chainID := cs1.Height, cs1.Round, cs1.state.ChainID
+	csT1, vssT1 := randState(4)
+	csT2, vssT2 := randStateWithAppPbts(4)
+
+	tcs := []struct {
+		name  string
+		state *State
+		vss   []*validatorStub
+		exp   error
+	}{
+		{name: "Without PBTS", state: csT1, vss: vssT1, exp: nil},
+		{name: "With PBTS", state: csT2, vss: vssT2, exp: nil},
```

Comment:
I don't understand whether the result `exp`, which is not even used , should change when running BFT Time or PBTS.

Reply:
exp is nil in both cases. I will remove it.